post-scan-actions/aws-python-conformity-custom-check/README.md

```diff
@@ -4,10 +4,13 @@ After a scan occurs and a malicious file is detected, this example Lambda functi
 
 ## Prerequisites
 
-1. **Configure Conformity API Key**
-    - Log into Conformity and select your username and select `User settings`.
-    - Select `API Keys`.
-    - Select `+ New API Key`, then select `Generate Key`.
+1. **Configure Clond One API Key for Conformity**
+    - In the upper-right corner of the Trend Micro Cloud One console, select the account where you want to add an API key.
+    - Click the account name and then `Account Settings`.
+    - Click on `API Keys` from the left hand side navigation.
```

Comment:
- On the left, click **API Keys**

Reply:
General comment: please use `**bold**` for user interface elements such as button names and field names.